README.md

```diff
@@ -5,53 +5,59 @@ The USDS website is built with:
 - [Jekyll](https://jekyllrb.com/) via the [github-pages gem](https://rubygems.org/gems/github-pages)
   - See [current dependencies](https://pages.github.com/versions/) for GitHub Pages
 - [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/)
-- [Gulp](https://gulpjs.com/) / [USWDS Gulp](https://github.com/uswds/uswds-gulp)
 
 Before getting started, install the following on your system:
 
-- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/), best set up on version 2.7 for Jekyll
 - [NPM](https://github.com/npm/cli)
-- [Gulp CLI](https://gulpjs.com/docs/en/getting-started/quick-start)
+- Docker
```

Comment:
Minor suggestion: this stands out as the only bullet that's not a link. Maybe link `https://www.docker.com/get-started/`    ?